examples/MintsMultipleTokens.js

```diff
@@ -0,0 +1,11 @@
+var rawData1 = require('Scripts/redeemer1.plutus');
```

Comment:
```suggestion
var rawData1 = require('scripts/redeemer1.plutus');
```

Reply:
Webpack aliases are usually capitalized in order to avoid conflicts with local folders

Reply:
Horrible, but OK.